mypy/options.py

```diff
@@ -305,6 +306,9 @@ def __init__(self) -> None:
         # -1 means unlimited.
         self.many_errors_threshold = defaults.MANY_ERRORS_THRESHOLD
 
+        # Prefix where mypy is installed, or None if default is used
+        self._prefix = None  # type: Optional[pathlib.Path]
```

Comment:
```suggestion
        self.prefix = None  # type: Optional[pathlib.Path]
```
Options objects are not prefixed with underscores.